x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/engine_overview_metrics.tsx

```diff
@@ -37,9 +37,14 @@ export const EngineOverviewMetrics: React.FC = () => {
         <UnavailablePrompt />
       ) : (
         <>
-          <TotalStats />
-          <EuiSpacer size="xl" />
-          <TotalCharts />
+          <EuiFlexGroup>
+            <EuiFlexItem grow={1}>
+              <TotalStats />
+            </EuiFlexItem>
+            <EuiFlexItem grow={3}>
+              <TotalCharts />
+            </EuiFlexItem>
+          </EuiFlexGroup>
```

Comment:
Renders the stats and charts in a row, rather than stacked vertically.

Reply:
Do we want to adjust/reduce the chart height the charts at all as a result? They look a little squashed on a laptop screen:

<img width="1143" alt="" src="https://user-images.githubusercontent.com/549407/109578369-eaaaf200-7aab-11eb-8c21-94628b041003.png">

Also there's some extra space that the current height is causing on the stat cards which we could easily lose:

<img width="318" alt="Screen Shot 2021-03-01 at 4 34 00 PM" src="https://user-images.githubusercontent.com/549407/109578464-0f06ce80-7aac-11eb-830c-5b64e25c678f.png">

Example with a `height={240}` passed into both AnalyticsCharts in the TotalCharts component:

<img width="1144" alt="" src="https://user-images.githubusercontent.com/549407/109578642-673dd080-7aac-11eb-8786-f159916246cb.png">

The above looks slightly more balanced to me - what do you think?

Reply:
Good call 👍🏼

Reply:
https://github.com/elastic/kibana/pull/92939/commits/9782526e982dd2ca32d8630efca8ef59d5d479fb